rubicon/objc/types.py

```diff
@@ -53,6 +53,172 @@ def get_ctype_for_type_map():
 _ctype_for_encoding_map = {}
 _encoding_for_ctype_map = {}
 
+def _end_of_encoding(encoding, start):
+    """Find the end index of the encoding starting at index start.
+    The encoding is not validated very extensively. There are no guarantees what happens for invalid encodings - an error may be raised, or a bogus end index may be returned.
```

Comment:
Nit: Can we break this into multiple lines? We try to follow at least the Django Style Guide (https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/)